crates/iota-genesis-builder/Cargo.toml

```
@@ -20,7 +20,7 @@ csv = "1.2"
 fastcrypto.workspace = true
 flate2.workspace = true
 itertools.workspace = true
-packable = { version = "0.8", default-features = false, features = ["io"] }
+packable = { workspace = true, features = ["io"], default-features = false }
```

```suggestion
packable = { workspace = true, features = ["io"] }
```
`default-features = false` in a member crate dependency has no effect and may be misleading. Default features are enabled/disabled in workspace manifest.